README.md

```diff
@@ -1,18 +1,18 @@
 # SalesBarChartGenerator
 Write a program that asks the user to enter today’s sales for five stores. The program should display a bar chart comparing each store’s sales. Create each bar in the bar chart by displaying a row of asterisks. Each asterisk should represent $100 of sales. Here is an example of the program’s output:
 
->Enter today's sales for store 1: 1000 [Enter]<br>
-Enter today's sales for store 2: 1200 [Enter]<br>
-Enter today's sales for store 3: 1800 [Enter]<br>
-Enter today's sales for store 4: 800 [Enter]<br>
-Enter today's sales for store 5: 1900 [Enter]<br>
-
->SALES BAR CHART<br>
-Store 1: **********<br>
-Store 2: ************<br>
-Store 3: ******************<br>
-Store 4: ********<br>
-Store 5: *******************<br>
+    Enter today's sales for store 1: 1000 [Enter]
+    Enter today's sales for store 2: 1200 [Enter]
+    Enter today's sales for store 3: 1800 [Enter]
+    Enter today's sales for store 4: 800 [Enter]
+    Enter today's sales for store 5: 1900 [Enter]
+    
+    SALES BAR CHART
+    Store 1: **********
+    Store 2: ************
+    Store 3: ******************
+    Store 4: ********
+    Store 5: *******************
```

Comment:
All I did was source format it and moved the commented statues. I didn't change anything else.

Sent from my iPhone

> On Mar 31, 2015, at 6:12 PM, Raffi Khatchadourian <notifications@github.com> wrote:
> 
> In README.md:
> 
> > -Store 2: ************<br>
> > -Store 3: ******************<br>
> > -Store 4: ********<br>
> > -Store 5: *******************<br>
> > +    Enter today's sales for store 1: 1000 [Enter]
> > +    Enter today's sales for store 2: 1200 [Enter]
> > +    Enter today's sales for store 3: 1800 [Enter]
> > +    Enter today's sales for store 4: 800 [Enter]
> > +    Enter today's sales for store 5: 1900 [Enter]
> > +    
> > +    SALES BAR CHART
> > +    Store 1: **********
> > +    Store 2: ************
> > +    Store 3: ******************
> > +    Store 4: ********
> > +    Store 5: *******************
> Why was this file changed?
> 
> —
> Reply to this email directly or view it on GitHub.
>